akka-testkit-typed/src/main/scala/akka/typed/testkit/StubbedActorContext.scala

```diff
@@ -109,23 +109,24 @@ private[typed] object WatchableRef {
  * provides only stubs for the effects an Actor can perform and replaces
  * created child Actors by a synchronous Inbox (see `Inbox.sync`).
  *
- * See [[EffectfulActorContext]] for more advanced uses.
+ * See [[BehaviorTestkit]] for more advanced uses.
  */
 class StubbedActorContext[T](
-  val name:                     String,
-  override val mailboxCapacity: Int,
-  override val system:          ActorSystem[Nothing]) extends ActorContextImpl[T] {
+  val name: String) extends ActorContextImpl[T] {
 
-  val selfInbox = Inbox[T](name)
+  val selfInbox = TestInbox[T](name)
   override val self = selfInbox.ref
+  override val system = new ActorSystemStub("StubbedActorContext")
+  // not used
+  override def mailboxCapacity = 1
```

Comment:
FIXME or remove

Reply:
It is part of the API so we have to over ride it, i'll make the comment more explicit this is the case